src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Compatibility.props

```diff
@@ -2,6 +2,6 @@
 <Project>
   <PropertyGroup>
     <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(MSBuildThisFileDirectory)..\tools\net472\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
-    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\netcoreapp3.1\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
+    <DotNetPackageValidationAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">$(MSBuildThisFileDirectory)..\tools\net6.0\Microsoft.DotNet.PackageValidation.dll</DotNetPackageValidationAssembly>
```

Comment:
It seems like you are missing to update the csproj to build for `net6.0` rather than `netcoreapp3.1`.

Reply:
Also, are these going to be under `tools` in the SDK? Is that how the SDK is composed?